tests/test_filter_params.py

```diff
@@ -0,0 +1,562 @@
+                    if model == SequencingCenter:
+                        data['name'] = 'SequencingCenter_{}'.format(i)
+                    m = model(**data)
+                    if model not in entities:
+                        entities[model] = []
+                    entities[model].append(m)
+
+                    db.session.add(m)
+
+            # Family relationships
+            for participant, relative in iterate_pairwise(
+                    entities[Participant]):
+                gender = participant.gender
+                rel = 'mother'
+                if gender == 'male':
+                    rel = 'father'
+                r = FamilyRelationship(participant=participant,
+                                       relative=relative,
+                                       participant_to_relative_relation=rel)
+                db.session.add(r)
+
+            # Cavatica task genomic files
+            for i, (ct, gf) in enumerate(zip(
+                    entities[CavaticaTask], entities[GenomicFile])):
+                is_input = True
+                if i % 2 == 0:
+                    is_input = False
+                ctgf = CavaticaTaskGenomicFile(cavatica_task=ct,
+                                               genomic_file=gf,
+                                               is_input=is_input)
+                db.session.add(ctgf)
+
+            # Add relations
+            s0 = entities[Study][0]
+            f0 = entities[Family][0]
+            p0 = entities[Participant][0]
+            sc0 = entities[SequencingCenter][0]
+            se0 = entities[SequencingExperiment][0]
+            ca0 = entities[CavaticaApp][0]
+
+            # Investigator
+            for inv in entities[Investigator]:
+                inv.studies.append(s0)
+            # Study file
+            for sf in entities[StudyFile]:
+                sf.study = s0
+            # Participant
+            for ent in entities[Participant]:
+                ent.study = s0
+                ent.family = f0
+
+            # Biospecimen, Diagnosis, Phenotype, Outcome
+            participant_ents = [Biospecimen, Diagnosis, Phenotype, Outcome]
+            for participant_ent in participant_ents:
+                for ent in entities[participant_ent]:
+                    ent.participant = p0
+                    if Biospecimen == participant_ent:
+                        ent.sequencing_center = sc0
+            # SequencingExperiment
+            for ent in entities[SequencingExperiment]:
+                ent.sequencing_center = sc0
+
+            # GenomicFiles
+            bs0 = entities[Biospecimen][0]
+            for ent in entities[GenomicFile]:
+                ent.sequencing_experiment = se0
+                ent.biospecimen = bs0
+
+            # CavaticaTask
+            for ent in entities[CavaticaApp]:
+                ent.cavatica_app = ca0
+
+            db.session.commit()
+
+    @pytest.mark.parametrize('model',
+                             [
```

Comment:
I think you could replace this list with `ENTITY_ENDPOINT_MAP.values()`

Reply:
yep 👍